documentation/common/attributes.adoc

```diff
@@ -76,7 +76,7 @@
 
 // Container image names and repositories
 :DockerOrg: docker.io/strimzi
-:DockerTag: {ProductVersion}
+:DockerTag: x.y.z
```

Comment:
The version is quite important in many of the places. I think we should keep this as it was.